docker-compose.yml

```diff
@@ -47,7 +47,7 @@ services:
     volumes:
       - postgres-data:/var/lib/postgresql/data
     ports:
-      - 5432:5432
+      - 5555:5555
```

Comment:
I think this'll need to be `5555:5432`, so it still references the right port inside the container, and we may also need to update the `GLOWFIC_DATABASE_HOST` env above to include the correct port.

However, if we're only reaching the port from within Docker, it might be worthwhile using [`expose`](https://docs.docker.com/compose/compose-file/#expose) instead?

Reply:
This could also be resolved by letting that mapping be arbitrary on the host and updating `bin-psql` to reflect that port